src/server/services/collection.service.ts

```diff
@@ -1297,22 +1386,34 @@ export const bulkSaveItems = async ({
 }) => {
   const collection = await dbRead.collection.findUnique({
     where: { id: collectionId },
-    select: { type: true, metadata: true, name: true },
+    select: { type: true, metadata: true, name: true, userId: true, mode: true },
   });
 
   if (!collection) throw throwNotFoundError('No collection with id ' + collectionId);
 
   const metadata = (collection.metadata ?? {}) as CollectionMetadataSchema;
 
-  await validateContestCollectionEntry({
-    metadata,
-    collectionId,
-    userId,
-    articleIds,
-    modelIds,
-    imageIds,
-    postIds,
-  });
+  if (collection.mode === CollectionMode.Contest) {
+    await validateContestCollectionEntry({
+      metadata,
+      collectionId,
+      userId,
+      articleIds,
+      modelIds,
+      imageIds,
+      postIds,
+    });
+  }
+
+  if (collection.userId == -1 && !collection.mode && collection.name.includes('Fe')) {
```

Comment:
I guess it should be this way
```suggestion
  if (collection.userId == -1 && !collection.mode && collection.name.includes('Featured')) {
```